staging/src/kubevirt.io/api/migrations/v1alpha1/types.go

```diff
@@ -161,7 +161,7 @@ func countMatchingLabels(policy *MigrationPolicy, vmiLabels, namespaceLabels map
 	countLabelsHelper := func(policyLabels, labelsToMatch map[string]string) {
 		for policyKey, policyValue := range policyLabels {
 			value, exists := labelsToMatch[policyKey]
-			if exists && value == policyValue {
+			if exists && (value == policyValue || policyValue == "") {
```

Comment:
So a blank policy is now an implicit wildcard?

@davidvossel any opinion on that?

Reply:
Not exactly.
Migration policies are being matched to VMIs during migration based on labels - either the VMI labels of the VMI's namespace labels.

So, let's say I want to match a certain policy to all VMIs that have label with key being `workload-type` and value being `db`, the migration policy spec would look like the following:
```yaml
  selectors:
    virtualMachineInstanceSelector:
      matchLabels:
        workload-type: db
```

As laid out in the [design proposal](https://github.com/kubevirt/community/blob/main/design-proposals/migration-policies.md), we should also support the case where we would like to match a policy to every VMI that has a label key `workload-type` but we don't care about the value. In this case the spec would look like that:

```yaml
  selectors:
    virtualMachineInstanceSelector:
      matchLabels:
        workload-type: ""
```

Reply:
for label selectors that is good. :+1: